src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments/Infrastructure/OuterApi/OuterApiService.cs

```diff
@@ -128,7 +128,19 @@ public async Task<long> CreateFileUploadLog(long providerId, IFormFile attachmen
             };
 
             var response = await _outerApiClient.Post<FileUploadLogResponse>(new PostFileUploadLogRequest(request));
-            return response.FileUploadLogId;
+            return response.LogId;
+        }
+
+        public async Task<long> UpdateFileUploadLog(long providerId, long logId)
+        {
+            var request = new FileUploadUpdateLogRequest
+            {
+                ProviderId = providerId,
+                LogId = logId,
+            };
+
+            var response = await _outerApiClient.Post<FileUploadUpdateLogResponse>(new PostFileUploadUpdateLogRequest(request));
+            return response.LogId;
```

Comment:
As above

Reply:
removed as part of update endpoint delete